### PR TITLE
Update health check script

### PR DESCRIPTION
**Pull request by ezynda3**

### CODE REVIEW

File: script/deploy/healthCheck.mts [hunk 1]
```diff
@@ -33,7 +32,6 @@ const corePeriphery = [
   'Receiver',
```
Comment (0xDEnYO) [inline script/deploy/healthCheck.mts:32]: I am wondering if it makes sense to read these from the config.sh (the config for our scripts) cause there we maintain this info already and it doesnt make so much sense to maintain it in different places. 
Alternatively we could consider to have subfolders "core" in our facets and periphery folder. Then we dont need to maintain the info anywhere but could just extract all names from the folder. What do you think?

File: script/deploy/healthCheck.mts [hunk 2]
```diff
@@ -356,6 +310,26 @@ const getOwnablContract = (address: Address, client: PublicClient) => {
   })
 }
 
+const checkOwnership = async (
+  name: string,
+  expectedOwner: Address,
+  deployedContracts: Record<string, Address>,
+  publicClient: PublicClient
+) => {
+  if (deployedContracts[name]) {
```
Comment (0xDEnYO) [inline script/deploy/healthCheck.mts:319]: I think I raised this before, not sure if you saw my comment. 
I feel that just reading from the deploy log can be dangerous. If a deployment goes wrong for some reason but the log is written anyway then we will not find this out.
I feel this script here should at least make sure that code is deployed at each contract address.

**Review by 0xDEnYO — APPROVED** [review]

File: script/deploy/healthCheck.mts (hunk 1 above)
Comment (ezynda3) [reply script/deploy/healthCheck.mts:32]: I agree with having a separate folder for core.

File: script/deploy/healthCheck.mts (hunk 2 above)
Comment (ezynda3) [reply script/deploy/healthCheck.mts:319]: Yes I plan to update this in a later PR

**Pull request merged**